CMakeLists.txt

```diff
@@ -375,7 +375,7 @@ message(STATUS "AMDGCN assembler: ${MIOPEN_AMDGCN_ASSEMBLER}")
 if(MIOPEN_USE_COMGR)
     find_package(amd_comgr REQUIRED CONFIG)
     message(STATUS "Build with comgr ${amd_comgr_VERSION}")
-    set(MIOPEN_PACKAGE_REQS "${MIOPEN_PACKAGE_REQS}, amd_comgr")
+    set(MIOPEN_PACKAGE_REQS "${MIOPEN_PACKAGE_REQS}, comgr")
```

Comment:
[Q] @junliume Does this work fine with ROCm 5.7.0 ?

Reply:
This issue is introduced by #2508 , not detected by CI because we do not have an "install deb" stage.